apps/prairielearn/src/pages/studentAssessmentInstance/studentAssessmentInstance.html.ts

```diff
@@ -45,6 +45,21 @@ export function StudentAssessmentInstance({
     }
   | { groupConfig?: undefined; groupInfo?: undefined; userCanAssignRoles?: undefined }
 )) {
+  let savedAnswers = 0;
+  let suspendedSavedAnswers = 0;
+  resLocals.instance_questions.forEach((question) => {
+    if (question.status === 'saved') {
+      if (question.allow_grade_left_ms > 0) {
+        suspendedSavedAnswers++;
+      } else if (question.max_auto_points) {
```

Comment:
This doesn't exactly do the right thing when `"points": [0], "manualPoints": 0` is used. In that case, "Save & grade" is still shown on the instance question page, but that question won't be gradable from the assessment instance page. Is there something better to check here, maybe `question.open && (question.max_auto_points || !question.max_manual_points)`? I think this mirrors the behavior from `lib/question-render.js`.

Reply:
That sounds plausible to me, but I'd appreciate @jonatanschroeder's thoughts on this too.

Reply:
Indeed the proper check based on points would be:
```suggestion
      } else if (question.max_auto_points || !question.max_manual_points) {
```
I believe if the status is 'saved', the question is presumably implied to be open. So a check for open may not be necessary in this context.

Reply:
Changed in 8acfdbf.